node/test/Test/Pos/Block/Logic/Mode.hs

```diff
@@ -110,7 +111,6 @@ import           Test.Pos.Block.Logic.Emulation (Emulation (..), runEmulation, s
 -- before testing starts.
 data TestParams = TestParams
     { _tpGenesisContext     :: !GenesisContext
-    -- ^ Genesis context.
```

Comment:
It does not convey any useful information. Are you sure?

Reply:
Yes, because later someone will write `TODO: comment` comment, then there will be wars, etc.
When there is a comment, nobody is concerned about adding it :)